cgmanifest.json

```diff
@@ -0,0 +1,74 @@
+{
+    "Registrations": [
+        {
+            "Component": {
+                "Pip": {
+                    "Name": "py",
+                    "Version": "1.9.0"
+                },
+                "Type": "pip"
+            },
+            "DevelopmentDependency": false
+        },
+        {
+            "Component": {
+                "Npm": {
```

Comment:
How come the npm dependencies aren't picked up by our package.json?

Reply:
Those are the alerts found here:
https://dev.azure.com/vscode-python-datascience/vscode-python-datascience/_componentGovernance/18591?_a=alerts&typeId=133049&alerts-view-option=active

I'm not sure why they aren't picked up automatically. Before you registered them right there, but now they're asking for repos to have that file.

The python repo also has it.

Reply:
I'm confused. They look like they are picked up to me. Otherwise how does the warning show up. I don't think this is going to fix the governance errors. You need to update our package.json to newer versions to update them.